test/Makefile

```diff
@@ -48,6 +56,21 @@ init-submodules:
 	git submodule update --init $(GTEST_MODULE)
 	touch init-submodules
 
+$(YAML_CPP_DIR)/Makefile: init-submodules
+	cd $(YAML_CPP_DIR) && \
+	$(CMAKE) -DMAKE_BUILD_TYPE=Debug -DCMAKE_CXX_FLAGS=-std=c++11 \
+	    -DMAKE_C_COMPILER=gcc -DCMAKE_CXX_COMPILER=g++ \
+	    -DYAML_CPP_BUILD_TOOLS=OFF
+
+$(YAML_CPP_LIBS): build-yaml-cpp
```

Comment:
Instead of copying all of this `make` cruft from `src/Makefile` here, why not delegate:
```Makefile
$(YAML_CPP_LIBS):
	cd $(SRC_DIR) && $(MAKE) $@
```
?

Reply:
Much better, changed.